modules/ocf/manifests/packages/gnupg.pp

```diff
@@ -0,0 +1,5 @@
+class ocf::packages::gnupg {
+  if $::lsbdistcodename != 'jessie' {
+    package { 'dirmngr':; }
+  }
+}
```

Comment:
seems fine to me, but perhaps leave a comment about why this is special or the reason will be forgotten eventually